### DISCUSSION TITLE
XRM Toolbox Disable override plugin folder

### DISCUSSION THREAD

**Discussion by mrussellMarlin** [MscrmTools/XrmToolBox#1233] [Q&A]
Hi,

Is  there a way to disable the override plugin folder command line switch?  We're trying to lock down XRM Toolbox for a secure environment and there are concerns that that command line switch could be used to let it run arbitrary code.

**Comment by MscrmTools** [comment]
This is not possible currently. This is something I can implement for future release

**Comment by fowl2** [comment]
A workaround/alternative/more general solution that would cover most .NET-based apps with plugins is the "Dynamic Code Security" feature of Windows Defender Application Control (WDAC), outlined [here](https://learn.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-with-dynamic-code-security).
